File: javascript2/Natural.js
var readln = (typeof readline === 'function') ? (readline) : (function() {
   importPackage(java.io);
   importPackage(java.lang);
   var stdin = new BufferedReader(new InputStreamReader(System['in']));
   return function() {
     return String(stdin.readLine());   
   };                                    
 }());
 print("How many natural numbers you want to print?")
 var Count = parseInt(readln())
var OutputString = "The natural numbers upto " + Count + " are"
for(Counter = 1; Counter <= parseInt(Count); Counter++) {
  OutputString = OutputString + " " + Counter 
  }
print(OutputString + ".")

File: Natural.java
import java.util.Scanner;
class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    System.out.print("Enter a number upto which you want to print natural numbers:");
    int Number = scanner.nextInt();
    System.out.print("The first " + Number + " natural numbers are");
    for(int Counter = 1; Counter <= Number; Counter++) {
      System.out.print(" " + Counter);
    }
    System.out.print(".");
  }
}